### ISSUE TITLE
Vertical scrolling features

### ISSUE THREAD

**Issue by mkruglikov** [thellmund/Android-Week-View#132]
Hi! Thanks for this library, I highly appreciate what you are doing here.

It would be great if you can add 2 features:
1. Add a vertical scroll listener. So we can hide/show the bottom navigation bar on scroll, for example.
2. Auto scroll to the first event in a day or at least the ability to scroll to a desirable hour.

Thanks!

**Comment by thellmund** [comment]
1. Good idea – I will take a look at that soon.
2. You can scroll to the desired hour via `weekView.goToHour(hour)`. You can also automatically scroll to the current hour on start-up via `app: showCurrentTimeFirst="true"` in the layout XML. Does that cover your use case?

**Comment by mkruglikov** [comment]
> 2\. Does that cover your use case?

Yes, it helped me. Thanks a lot

**Comment by dsounded** [comment]
+1 for vertical scroll listeners

**Comment by dsounded** [comment]
because if we wanna use `SwipeRefreshLayout` then we cannot scroll to the 00 AM hour since the refresh is always called, that will be good to disable/enable swipe refresh only if current position(hour) is on the top

**Comment by dsounded** [comment]
It appears it can be done with `.firstVisibleHour` property

**Comment by thellmund** [comment]
Will have a look at this once #215 is merged. #213 is in, which will help.

**Comment by imkhalid** [comment]
goToHour or scrollToTime not work instead it scrolls to max hour. ?

**Comment by thellmund** [comment]
Please open a new issue if there is a bug and include a link to a sample project that showcases it.
